packages/subgraph/tests/constants.ts

```diff
@@ -50,6 +53,16 @@ export const CREATED_AT = '1644850000';
 
 export const ZERO_BYTES32 =
   '0x0000000000000000000000000000000000000000000000000000000000000000';
+export const ONE_BYTES32 =
+  '0x0000000000000000000000000000000000000000000000000000000000000001';
+export const HALF_UINT256_BYTES32 =
+  '0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
+export const MAX_UINT256_BYTES32 =
+  '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
+
+export const MAX_UINT256_NUMBER_STRING =
```

Comment:
why not having it as `BigInt` type here ? why string ? up to you. I don't know the exact thing.

Reply:
I want to create all bigInts from string and then check the function `bigIntToBytes32` result against expected result, for testing purposes